scripts/core/world/world.js

```diff
@@ -2,42 +2,60 @@ define([
         'lodash',
         '../random.js',
         '../../util/world-parameters.js',
+        '../localization/location.js',
         './world-status.js',
     ],
-    function(_, Generator, Parameters, WorldStatus) {
+    function(_, Generator, Parameters, Location, WorldStatus) {
         var World = function(seed, width, height) {
-            this.width = width;
-            this.height = height;
-            this.world = {};
-            this.generator = new Generator(seed);
+            var myself = this;
+            var tiles = {};
+            var generator = new Generator(seed);
 
             this.getWidth = function() {
-                return this.width;
+                return width;
             };
 
             this.getHeight = function() {
-                return this.height;
+                return height;
+            };
+
+            var initializeRow = function(rowIndex) {
+                _.times(width, function(w) {
+                    myself.getWorldStatus(new Location(w, rowIndex));
+                })
+            };
+
+            var initializeWorld = function() {
+                _.times(height, function(h) {
+                    initializeRow(h);
+                })
             };
 
             this.getWorldStatus = function(location) {
```

Comment:
Thats the method you ll need to call from the world object and giving it a location ( new Location(x,y) ) to get a WorldStatus object which will tell you if this location is dry / warm / wet / etc...